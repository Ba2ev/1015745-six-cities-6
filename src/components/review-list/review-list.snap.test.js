import React from 'react';
import {render} from '@testing-library/react';
import ReviewList from './review-list';

const reviews = [
  {
    "id": 1,
    "user": {
      "id": 18,
      "isPro": true,
      "name": `Sophie`,
      "avatarUrl": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/9.jpg`
    },
    "rating": 2,
    "comment": `Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.`,
    "date": `2021-03-07T08:04:28.647Z`
  },
  {
    "id": 2,
    "user": {
      "id": 15,
      "isPro": false,
      "name": `Kendall`,
      "avatarUrl": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/6.jpg`
    },
    "rating": 4,
    "comment": `What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!`,
    "date": `2021-03-07T08:04:28.647Z`
  }
];

it(`ReviewList should render correctly`, () => {
  const {container} = render(<ReviewList reviews={reviews}/>);

  expect(container).toMatchSnapshot();
});
