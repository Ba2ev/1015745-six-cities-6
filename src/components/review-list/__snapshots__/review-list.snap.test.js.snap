// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewList should render correctly 1`] = `
<div>
  <ul
    class="reviews__list"
  >
    <li
      class="reviews__item"
    >
      <div
        class="reviews__user user"
      >
        <div
          class="reviews__avatar-wrapper  user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            class="reviews__avatar user__avatar"
            height="54"
            src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/9.jpg"
            width="54"
          />
        </div>
        <span
          class="reviews__user-name"
        >
          Sophie
        </span>
      </div>
      <div
        class="reviews__info"
      >
        <div
          class="reviews__rating rating"
        >
          <div
            class="reviews__stars rating__stars"
          >
            <span
              style="width: 40%;"
            />
            <span
              class="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          class="reviews__text"
        >
          Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.
        </p>
        <time
          class="reviews__time"
          datetime="2021-03-07"
        >
          March 2021
        </time>
      </div>
    </li>
    <li
      class="reviews__item"
    >
      <div
        class="reviews__user user"
      >
        <div
          class="reviews__avatar-wrapper  user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            class="reviews__avatar user__avatar"
            height="54"
            src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/6.jpg"
            width="54"
          />
        </div>
        <span
          class="reviews__user-name"
        >
          Kendall
        </span>
      </div>
      <div
        class="reviews__info"
      >
        <div
          class="reviews__rating rating"
        >
          <div
            class="reviews__stars rating__stars"
          >
            <span
              style="width: 80%;"
            />
            <span
              class="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          class="reviews__text"
        >
          What an amazing view! The house is stunning and in an amazing location. The large glass wall had an amazing view of the river!
        </p>
        <time
          class="reviews__time"
          datetime="2021-03-07"
        >
          March 2021
        </time>
      </div>
    </li>
  </ul>
</div>
`;
